package-lock.json

```diff
@@ -567,13 +569,67 @@
       "name": "@keymanapp/ldml-keyboard-constants",
       "license": "MIT"
     },
+    "developer/src/common/web/test-helpers": {
+      "name": "@keymanapp/developer-test-helpers",
+      "license": "MIT",
+      "dependencies": {
+        "@keymanapp/common-types": "*"
+      },
+      "devDependencies": {
+        "@types/node": "^10.14.6",
```

Comment:
Should this be the same version as l. 585? 10.17.60

Reply:
Don't review package-lock.json -- it's auto generated. Look only at the various package.json files :grin: